.github/workflows/build.yml

```diff
@@ -71,3 +71,14 @@ jobs:
           target: "${{ secrets.DIVE_STAGING_DIRECTORY }}"
           strip_components: 3
 
+      - name: Deploy ais-staging
+        uses: appleboy/scp-action@v0.0.10
+        with:
+          host: ${{ secrets.SERVER }}
+          username: ${{ secrets.USER }}
+          key: ${{ secrets.SSH_PRIVATE_KEY }}
+          rm: true
+          source: "dist/apps/ais"
+          target: "${{ secrets.ais_STAGING_DIRECTORY }}"
```

Comment:
Make it `AIS_STAGING_DIRECTORY`

Reply:
I change it to Make it AIS_STAGING_DIRECTORY